target/linux/ath79/image/generic-ubnt.mk

```diff
@@ -108,6 +108,14 @@ define Device/ubnt_rocket-m
 endef
 TARGET_DEVICES += ubnt_rocket-m
 
+define Device/ubnt_nanostation-loco-m-xw
+  $(Device/ubnt-xw)
+  DEVICE_MODEL := Nanostation Loco M
+  DEVICE_VARIANT := XW
```

Comment:
Since ynezz just pulled my PR #2425 , you can remove this line as it will be inherited from ubnt-xw.

Reply:
Line "DEVICE_VARIANT := XW" removed.